gcp/table_gcp_kms_key.go

```diff
@@ -74,6 +74,11 @@ func tableGcpKmsKey(ctx context.Context) *plugin.Table {
 				Hydrate:     getKeyIamPolicy,
 				Transform:   transform.FromValue(),
 			},
+			{
+				Name:        "labels",
+				Description: "Labels that apply to this kms key.",
```

Comment:
@bigdatasourav Where is this description from? Is it from the SDK, API, or somewhere else?